politeiawww/cmd/politeiavoter/politeiavoter.go

```diff
@@ -762,14 +763,28 @@ func (c *ctx) dumpTogo() {
 
 // signalHandler catches SIGUSR1 and dumps the current state of the vote
 // trickler.
-func (c *ctx) signalHandler(signals chan os.Signal, done chan struct{}) {
+func (c *ctx) signalHandler(signals chan os.Signal, done chan struct{}, interrupt chan os.Signal, token string) {
 	for {
 		select {
 		case <-signals:
 			fmt.Printf("----- politeiavoter status -----\n")
 			c.dumpTogo()
 			c.dumpComplete()
 			c.dumpQueue()
+		case <-interrupt:
```

Comment:
Sample print out:
```
----- 707c7c6320165529d489faed71a5f045e7930dbba74f3540eb67a4d0ca21cc3e vote status -----
{
  "token": "707c7c6320165529d489faed71a5f045e7930dbba74f3540eb67a4d0ca21cc3e",
  "status": 3,
  "totalvotes": 1,
  "optionsresult": [
    {
      "option": {
        "id": "yes",
        "description": "Don't approve proposal",
        "bits": 1
      },
      "votesreceived": 1
    },
    {
      "option": {
        "id": "no",
        "description": "Approve proposal",
        "bits": 2
      },
      "votesreceived": 0
    }
  ],
  "endheight": "494718",
  "bestblock": "492706",
  "numofeligiblevotes": 6308,
  "quorumpercentage": 20,
  "passpercentage": 60
}
```

This is printing the overall vote stats of the proposal.  We want it to print the vote stats of the individual vote command.  Let's say you're trickling in 200 votes.  You'd want to know information like how many were cast, how many failed, how many are remaining.

We also want this to be printed in a nice, human readable format instead of just the json.  Kinda like the inventory command printout.

```
Vote: 707c7c6320165529d489faed71a5f045e7930dbba74f3540eb67a4d0ca21cc3e
  Proposal        : e44e4fe21332a65e
  Start block     : 492686
  End block       : 494718
  Mask            : 3
  Eligible tickets: 164
  Eligible votes  : 163
  Vote Option:
    Id                   : yes
    Description          : Don't approve proposal
    Bits                 : 1
    To choose this option: politeiavoter vote 707c7c6320165529d489faed71a5f045e7930dbba74f3540eb67a4d0ca21cc3e yes
  Vote Option:
    Id                   : no
    Description          : Approve proposal
    Bits                 : 2
    To choose this option: politeiavoter vote 707c7c6320165529d489faed71a5f045e7930dbba74f3540eb67a4d0ca21cc3e no
```